src/components/reusuables/cards/TeamCard.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { StaticImage } from "gatsby-plugin-image"
+import { Col } from "react-bootstrap"
+
+const TeamCard = (props) => {
```

Comment:
Destructure props like the last review.
(props) should be ({heading, subtitle, etc})